learn/getting_started/quick_start.mdx

```diff
@@ -45,27 +50,48 @@ Database path:       "./data.ms"
 Server listening on: "localhost:7700"
 ```
 
-Congratulations! You're ready to move on to the next step!
+You now have a Meilisearch instance running in your terminal window. Keep this window open for the rest of this tutorial.
+
+<Capsule intent="note">
+The above command uses the `--master-key` configuration option to secure Meilisearch. Setting a master key is optional but strongly recommended in development environments. Master keys are mandatory in production environments.
+</Capsule>
+
+To learn more about securing Meilisearch, refer to the [master and API keys guide](/learn/security/master_api_keys).
 
 ## Add documents
 
-For this quick start, we will be using a collection of movies as our dataset. To follow along, first click this link to download the file: <ButtonLink as="a" id="downloadMovie" href="/movies.json" download="movies.json">movies.json</ButtonLink>. Then, move the downloaded file into your working directory.
+In this quick start, you will search through a collection of movies.
+
+To follow along, first click this link to download the file: <ButtonLink as="a" id="downloadMovie" href="/movies.json" download="movies.json">movies.json</ButtonLink>. Then, move the downloaded file into your working directory.
+
+<Capsule intent="note">
+Meilisearch accepts data in JSON, NDJSON, and CSV formats.
+</Capsule>
 
 Open a new terminal window and run the following command:
 
 <CodeSamples id="getting_started_add_documents_md" />
 
-Meilisearch stores data in the form of discrete records, called [documents](/learn/core_concepts/documents). Documents are grouped into collections, called [indexes](/learn/core_concepts/indexes).
+Meilisearch stores data in the form of discrete records, called [documents](/learn/core_concepts/documents). Each document is composed of multiple fields, which are pairs of one attribute and one value:
 
-<Capsule intent="note">
-Meilisearch currently only accepts data in JSON, NDJSON, and CSV formats. You can read more about this in our [documents guide](/learn/core_concepts/documents#dataset-format).
-</Capsule>
+```json
+{
+  "id": 0,
+  "attribute": "value"
```

Comment:
I guess you're right. I thought it could be good to show the id attribute, but now you mention it `id` might distract users from the main point we're trying to illustrate here.